src/modules/dashboard/components/ColonyHome/ColonyTitle/ColonyTitleMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { Colony } from '~data/index';
+import MaskedAddress from '~core/MaskedAddress';
+import InvisibleCopyableAddress from '~core/InvisibleCopyableAddress';
+
+import styles from './ColonyTitle.css';
+
+const MSG = defineMessages({
+  copyMessage: {
+    id: 'dashboard.ColonyHome.ColonyTitle.copyMessage',
+    defaultMessage: 'Click to copy colony address',
+  },
+});
+
+type Props = {
+  colony: Colony;
+  children?: React.ReactChild;
+};
+
+const displayName = 'dashboard.ColonyHome.ColonyTitle';
+
+const ColonyTitleMenu = ({ colony: { colonyAddress }, children }: Props) => {
```

Comment:
Didn't you add this already in https://github.com/JoinColony/colonyDapp/pull/3485 ?